developer/src/common/web/utils/src/xml-utils.ts

```diff
@@ -0,0 +1,122 @@
+import { xml2js } from "./index.js";
+
+export class KeymanXMLOptions {
+    type: 'keyboard3'         // LDML <keyboard3>
```

Comment:
I think this should be declared as a type, so it can be used in my other comments